go.mod

```diff
@@ -4,9 +4,12 @@ go 1.22.0
 
 toolchain go1.22.4
 
+// Later versions break the build of the os provider for Windows
+replace modernc.org/libc => modernc.org/libc v1.53.3
```

Comment:
The build works now:
https://github.com/mondoohq/cnquery/actions/runs/9888135455

Reply:
Nice!